tests/foreman/ui/test_subscription.py

```diff
@@ -483,3 +483,57 @@ def test_positive_subscription_status_disabled(session, content_host_setup):
                 'subscription_status'
             ]
             assert "Disabled" in host
+
+
+@tier2
+def test_positive_candlepin_events_processed_by_STOMP(session):
+    """Verify that Candlepin events are being read and processed by
+       attaching subscriptions, validating host subscriptions status,
+       and viewing processed and failed Candlepin events
+
+    :id: 9510fd1c-2efb-4132-8665-9a72273cd1af
+
+    :steps:
+
+        1. Register Content Host without subscriptions attached
+        2. Verify subscriptions status is red "invalid"
+        3. Import a Manifest
+        4. Attach subs to content host
+        5. Verify subscription status is green "valid"
+        6. Check for processed and failed Candlepin events
+
+    :expectedresults: Candlepin events are being read and processed
+                      correctly without any failures
+
+    :BZ: #1826515
+
+    :CaseImportance: High
+    """
+    org = entities.Organization().create()
+    repo = entities.Repository(product=entities.Product(organization=org).create()).create()
+    repo.sync()
+    ak = entities.ActivationKey(
+        content_view=org.default_content_view,
+        max_hosts=100,
+        organization=org,
+        environment=entities.LifecycleEnvironment(id=org.library.id),
+    ).create()
```

Comment:
Just suggestion, Can we write or reuse the fixture for the basic setup?

Reply:
I think since this is a unique case of registering a host without subs, it isn't necessary to create a fixture.